### DISCUSSION TITLE
MobileStart Bypass/Always On

### DISCUSSION THREAD

**Discussion by ISimion** [astrandb/miele#417] [Q&A] [answered]
I bought a **Washer/Dryer WTR870WPM** and successfully connected my appliance through Miele@Home technology on my **Miele App - Smart Home**. However, I must start the machine and select the **MobileStart** option whenever I want to control it from the app. 

My question is: 

1. Would it be possible to program it so I could control the machine from the app only by starting the machine and **bypassing** the MobileStart option (or the MobileStart option **always** to be **on**)?
2. Even better, could I start my washing machine remotely straight from the app?

**Comment by mundschenk-at** [comment]
This hasn't got anything to with the integration, has it? You'd have to ask Miele those questions (though the answer is very likely "no").

**Reply by ISimion** [reply]
I asked here to see if anybody knows if Miele API version 1.0.7 would be able to offer control over the MobileStart option on the washing machine. If there can be an API call to activate/deactivate that option.

**Reply by astrandb** [reply, marked as answer]
No, the Mobile Start option is read-only in the API.
